### PR TITLE
Security Baseline for Windows 10 v1909 is Final

### PR DESCRIPTION
**Pull request by paselm**
i have added the Information for Windows 10 v1909.

### CODE REVIEW

File: windows/security/threat-protection/security-compliance-toolkit-10.md [hunk 1]
```diff
@@ -35,7 +36,7 @@ The Security Compliance Toolkit consists of:
     -   Windows 10 Version 1507
 
 -   Windows Server security baselines
-    -   Windows Server 2019
+    -   Windows Server 2019
```
Comment (illfated) [inline windows/security/threat-protection/security-compliance-toolkit-10.md]:
```suggestion
    -   Windows Server 2019
```
(redundant whitespace added -- no need)

**Comment by illfated** [comment]
Also, please note that someone else has already opened a PR for this change: #5497 (**Update security-compliance-toolkit-10.md**)

**Comment by illfated** [comment]
Now this is quite odd. I was expecting a file conflict here between this PR and #5497, but for some odd reason, this has not happened.

Path+filename:
- Content featured in this PR: branch:  [`MicrosoftDocs:public`](https://github.com/MicrosoftDocs/windows-itpro-docs) -- path: windows/security/threat-protection/security-compliance-toolkit-10.md
- https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/security-compliance-toolkit-10.md PR #5497 https://github.com/MicrosoftDocs/windows-itpro-docs/commit/8937e217ac3525e1851328d56e2a66c64f91a258

**Comment by e0i** [comment]
Duplicate of #5497 (Merged)

**Pull request closed without merging**